docs: Add Redpanda Cloud to the quickstart (#1260)
redpanda-data/documentation: modules/reference/pages/public-metrics-reference.adoc

---

=== redpanda_cluster_latest_cluster_metadata_manifest_age

The amount of time in seconds since the last time Redpanda uploaded metadata files to Tiered Storage for your cluster. A value of `0` indicates metadata has not yet been uploaded.

> When performing a whole cluster restore operation, the metadata for the new cluster will not have any changes made to a source cluster that is newer than this age.

Reviewer: ```suggestion
When performing a whole cluster restore operation, metadata for new clusters will not include any changes made to a source cluster that is newer than this age.
```

[[data_transform_metrics]]
== Data transforms metrics

=== redpanda_transform_execution_errors

Counter for the number of errors encountered during the invocation of data transforms.

*Type*: counter

*Labels*:

* `function_name`

---

=== redpanda_transform_execution_latency_sec

> Histogram tracking the execution latency (in seconds) for processing a single record via data transforms.

Reviewer: ```suggestion
Histogram tracking the execution latency (in seconds) for processing a single record using data transforms.
```

---

=== redpanda_wasm_binary_executable_memory_usage

> Amount of memory (in bytes) used by executable WebAssembly binaries.

Reviewer: ```suggestion
Number of bytes (memory) used by executable WebAssembly binaries.
```

Reviewer: Interesting one:
The terms "amount" and "number" both refer to quantity but are used in different contexts:
Amount is used with uncountable nouns (mass nouns) such as water, sand, or information. For example, you would say "a large amount of water."
[Number is used with countable nouns (count nouns) such as apples, cars, or people. For example, you would say "a large number of apples". ](https://writingexplained.org/amount-vs-number-difference)
[3](https://writingexplained.org/amount-vs-number-difference)

[In summary, use "amount" for things that cannot be counted individually and "number" for things that can be counted.](https://www.grammarly.com/commonly-confused-words/amount-vs-number)
